owsi-core/owsi-core-components/owsi-core-component-wicket-more/src/test/java/fr/openwide/core/test/wicket/more/model/TestSuiteGenericEntityCollectionModel.java

```diff
@@ -42,7 +42,7 @@ public TestGenericEntityArrayListModel() {
 
 		@Override
 		protected IModel<List<Person>> createModel() {
-			return GenericEntityArrayListModel.of(Person.class);
+			return CollectionCopyModel.custom(Suppliers2.<Person>arrayListAsList(), GenericEntityModel.<Person>factory());
```

Comment:
There already are tests for `CollectionCopyModel`s in `TestSerializableCollectionCopyModel` and  `TestGenericEntityCollectionCopyModel`.
If you removed `GenericEntityArrayListModel`, this test (`TestSuiteGenericEntityCollectionModel`) should have gone away. If you simply deprecated `GenericEntityArrayListModel`, then the tests should have stayed the same (since `GenericEntityArrayListModel` is still used, we still want to  ensure it works properly). Same goes for other models.
Could you please fix that?